Drift/Public/Drift.swift

```diff
@@ -19,6 +19,7 @@ open class Drift: NSObject {
      
     */
     open class func setup(_ embedId: String) {
+        DriftDataStore.sharedInstance.setEmbedId(embedId)
```

Comment:
Do we want to do this? In the retrieve data from embeds call we check to make sure the embed is the same as whats in the data store to drop all the caches and start again.